### ISSUE TITLE
An issue on Resume Training

### ISSUE THREAD

**Issue by ruili3** [sconlyshootery/FeatDepth#36]
Hi,

I have a question on resuming training on previous trained models. As my GPU server constrains the running time for each job, I have to resume the training process to get the desired epoch (40). I set the 'resume' parameter to the model path of previous epoch. 

Under this circumstance, I wonder is there other parameters I should change for the taining? (i.e. the learning rate? I wish to use the learning rate set for the previous epoch. ) Should I change the parameter 'total_epochs ' ? Should the number '1' in 'workflow = [('train', 1)]' be reset to the number of the next epoch?

Thank you very much!

**Comment by sconlyshootery** [comment]
In the resume mode, the settings will be kept the same with the previous settings saved in ckpt, so any change in the cfg will not work.
If you want to use new settings, you can use finetune mode, any change in the cfg will work.

**Comment by ruili3** [comment]
Yes, the resume mode is exactly what I want, so I just need to set
'resume_from = ' to the model path, and keep 'finetune=' to None?

sconly <notifications@github.com> 于2021年2月18日周四 上午11:07写道：

**Comment by sconlyshootery** [comment]
Yes, that is right.
